src/webots/editor/WbProjectRelocationDialog.cpp

```diff
@@ -224,67 +224,29 @@ void WbProjectRelocationDialog::copy() {
   setStatus(tr("Project successfully relocated.") + "\n" + tr("%1 file(s) copied.").arg(copiedFilesCount));
 }
 
-int WbProjectRelocationDialog::copyProject(const QString &projectPath, bool copyProtoProject) {
-  // Copy modified project resources:
-  // current project folder or project folder of an external PROTO model used in the current project.
-  // When modifying any resource (world file, PROTO file, controller, plugins, etc.) all the other resources stored in
-  // the current project path have to also be copied. Only non-modified PROTO related files stored in the standard paths can be
-  // skipped.
-  //
-  // Here we should copy only the files that may be edited / changed by the user, that is:
-  // 1) textures, meshes, and skins in `protos` folders if PROTO file was modified
-  // 2) all the controllers (folders) and related libraries that was modified or used by this world and which are not located in
-  //    the current project folder. Skip controllers used by PROTO models that have not been modified.
-  // 3) possibly the current controller file (even if corresponding to a PROTO robot)
-  // 4) project libraries if a controller or plugin is copied
-  // 5) project motions folder if a controller is copied
-
+int WbProjectRelocationDialog::copyProject(const QString &projectPath) {
   int result = 0;
 
-  if (copyProtoProject) {
-    // copy all PROTO textures
-    result += WbFileUtil::copyDir(projectPath + "protos/textures", mTargetPath + "/protos/textures", true, true, true);
-    result += WbFileUtil::copyDir(projectPath + "protos/meshes", mTargetPath + "/protos/meshes", true, true, true);
-    result += WbFileUtil::copyDir(projectPath + "protos/skins", mTargetPath + "/protos/skins", true, true, true);
-  }
-
-  bool projectLibrariesCopied = false;
+  // copy all local resources
+  result += WbFileUtil::copyDir(projectPath + "protos", mTargetPath + "/protos", true, false, true);
```

Comment:
More of a question really, does the project relocation action trigger only for our own folders? If so can't we skip copying the protos folder? It should be empty no? (well, technically it shouldn't be there to begin with after the cleanup of the distribution)

Reply:
Yes, good point. I can remove this copy of this `protos` folder indeed.

Reply:
Finally, I would prefer to keep it like that for the following reasons:
1. At some point we may include a protos folder in our example to demonstrate how to have a local PROTO.
2. Some people may fork Webots and add their own projects which may include a protos folder.
3. Some people may distribute some project folder which installs without write rights, and thus users would need to relocate it, including possibly a protos folder.